src/app/components/wait/index.html

```diff
@@ -51,6 +51,12 @@
     >
     Starts a non-blocking page wait on the page.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="blockingWrap"
+      isOptional="true"
+    >
+    Launches a page wait and automatically stops when the specific asynchronous event completes. The most common use case includes backend calls.
```

Comment:
For consistency with the rest of the descriptions, I think we need "blocking" at the start: "Launches a blocking page wait..."

Also, is the last sentence correct? Just making sure that the most common use case for blocking page waits is the same as for non-blocking page waits since the last sentence of that method is identical to the one here.

Reply:
I believe Colby contributed this so I checked in with him when adding the properties. All that to say, the second line doesn't really match existing docs so I'm okay to leave out the second sentence.

Reply:
Well, if the most common use case is the same for both and it is useful to point it out, I think it's fine to keep that sentence. I just wasn't sure if the use case would be the same for both. Thanks!